greedy_max/schedule.py

```diff
@@ -164,11 +164,11 @@ def sky_constraints(self) -> None:
         Use the most restrictive value for each condition. 
         '''
         restrictive_iq = min([obs.sky_conditions.iq for obs in self.observations])
-        restrictive_bg = min([obs.sky_conditions.sb for obs in self.observations])
+        restrictive_sb = min([obs.sky_conditions.sb for obs in self.observations])
         restrictive_cc = min([obs.sky_conditions.cc for obs in self.observations])
         restrictive_wv = min([obs.sky_conditions.wv for obs in self.observations])
 
-        return SkyConditions(restrictive_iq,restrictive_bg,restrictive_cc,restrictive_wv)
+        return SkyConditions(restrictive_sb,restrictive_cc,restrictive_iq,restrictive_wv)
```

Comment:
Not to nitpick. but to get rid of warnings, we should put spaces after commas: it's a Python standard.